intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -61,7 +61,7 @@ cd build
 cmake ..
 cmake --build . -j
 ```
-
+Note: add compile args ```-DNE_AVX512=OFF -DNE_AVX512_VBMI=OFF -DNE_AVX512_VNNI=OFF``` to ```cmake``` when compiling it on a CPU without AVX512
```

Comment:
could we auto detect the machine's isa to add follow compile args without manually ?

Reply:
fix in [PR511](https://github.com/intel/intel-extension-for-transformers/pull/511)

Reply:
I don't think so. If I remember correctly, there was a similar discussion for the deprecated executor, and the result is that it is not possible to find the hardware info of the runtime match during compile time on the machine that compiles the executable.
cc @luoyu-intel